src/config-loader/config-loader.ts

```diff
@@ -195,8 +195,15 @@ export const setup = (app: Express.Application) => {
 						);
 
 						const apiRoute = `/${model.apiRoot}/*`;
-						app.options(apiRoute, (_req, res) => res.status(200).end());
-						app.all(apiRoute, sbvrUtils.handleODataRequest);
+						app
+							.route(apiRoute)
+							.get(sbvrUtils.handleODataRequest)
+							.put(sbvrUtils.handleODataRequest)
+							.post(sbvrUtils.handleODataRequest)
+							.patch(sbvrUtils.handleODataRequest)
+							.merge(sbvrUtils.handleODataRequest)
+							.delete(sbvrUtils.handleODataRequest)
+							.options((_req, res) => res.status(200).end());
```

Comment:
I'd prefer to put `options` first in the list as it is a pre-flight request and hence will always be the first request to come in (if it does) so it feels like the right place to put it/is where I would expect it to be and look for it first

Reply:
I see, will put it on top of the list. Thanks